exporter/collector/googlemanagedprometheus/naming_test.go

```diff
@@ -45,16 +45,6 @@ func TestGetMetricName(t *testing.T) {
 		expected  string
 		expectErr bool
 	}{
-		{
-			desc:     "sum without total",
-			baseName: "foo",
-			metric: func(m pmetric.Metric) {
-				m.SetName("foo")
-				sum := m.SetEmptySum()
-				sum.SetIsMonotonic(true)
-			},
-			expected: "foo/counter",
```

Comment:
I think we should keep this test, just update the expected to show that a sum without total will get _total appended

Reply:
renamed and added it back